app/views/presentations/show.html.erb

```diff
@@ -10,3 +10,9 @@
   <dt>Date</dt>
   <dd><%= @presentation.event.date %></dd>
 </dl>
+
+<%= link_to 'Edit', edit_presentation_path(@presentation) %>
+<%= link_to 'Delete', presentation_path(@presentation),
+  method: :delete,
+  data: { confirm: 'Are you sure?' } %>
```

Comment:
I'd kind of like to see these parameters lined up a little more but if rubocop is alright with it then I am too.